test/static-code

```diff
@@ -48,6 +48,16 @@ test_pycodestyle() {
     find_python_files | xargs -r -0 python3 -m pycodestyle --max-line-length=415 --ignore W504
 }
 
+test_pyvulture() {
+    # vulture to find unused variables/functions
+
+    python3 -c 'import vulture' 2>/dev/null || skip 'no python3-vulture'
+    find_python_files | xargs -r -0 python3 -m vulture \
+        --min-confidence "${VULTURE_CONFIDENCE:-100}" \
+        --ignore-names 'do_*,test[A-Z0-9]*,__*__' \
```

Comment:
Are the `do_*()` methods still something to clean up after that PR, just too hard to do it right away? Or is there a reason why they can't be fixed?

Reply:
Note that none of these ignore flags affect anything at the 100% confidence level: you need to turn it down to 60% confidence before you'd ever see something affected by that . As for the specific case of `do_`: this is a semi-convention for virtual functions involved by another framework.  This is in use by at least gobject-introspection (for the GApplication vfuncs implemented in CockpitClient) as well as by the `do_GET`, `do_POST`, etc. methods called by the Python http server code.

The convention is strong enough that vulture even lists it as an example:

```
  --ignore-names PATTERNS
                        Comma-separated list of names to ignore (e.g., "visit_*,do_*").
```

but doesn't enable it by default, unfortunately.